src/js/components/addData/AddDataMeta.jsx

```diff
@@ -49,6 +49,14 @@ export default class AddDataMeta extends React.Component {
         };
     }
 
+    componentWillMount(){
+        this.setState({isMounted: true})
```

Comment:
Remember `isMounted` had problems. Everything else uses `isUnmounted`, we should stay consistent.